src/components/blocks/complex-functionality/Input.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { HtmlBlockComponentProps } from '../block';
+import SelfClosingHtmlBlock from '../Html/SelfClosingHtmlBlock';
+import './styles.css';
+
+const Input = ({ attribs }: HtmlBlockComponentProps) => {
```

Comment:
Remove `HtmlBlockComponentProps` and type Input as `Input: FC<HtmlBlockComponentProps>`. It is what all the cool kids do.